support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -1232,6 +1232,19 @@ export function CheckoutComponent({
 								})}
 							</RadioGroup>
 						</FormSection>
+						<div
+							css={css`
+								margin: ${space[6]}px 0;
+							`}
+						>
+							{abParticipations.similarProductsConsent === 'VariantB' && (
```

Comment:
Should we also check the product here and omit this if it's a product which shouldn't ever trigger the consent?

Reply:
I was going to do this in a follow up PR because the changes in this one are already quite extensive but I'm happy to add it here if you think that's better?

Reply:
Ah okay that makes sense, I guess no-one will see this in the meantime because the test isn't active.